src/components/BounceRulesContainer/index.jsx

```diff
@@ -184,7 +144,8 @@ const BounceRulesContainer = ({
             </div>
           )}
           {!isRulesEmpty &&
-            !isFetching && (
+            !isFetching &&
```

Comment:
not a must, but I find it makes it more readable when you have a bunch of checks in a conditional to wrap it all in a descriptive variable. for example, here I would wrap `!isRulesEmpty && !isFetching &&  isBounceRulesTab ` into `shouldShowBounceRules` and use `shouldShowBounceRules` here instead. same comment for all of these conditional checks here in this file